tests/specs/ecosystem/core/match/0_1_5.test

```diff
@@ -18,6 +18,15 @@ core/match/0.1.5
 
 ===
 
-== FAST CHECK EMIT PASSED ==
+== FAST CHECK EMIT FAILED ==
+error[unsupported-expando-property]: expando property referencing 'greedy' conflicts with 'placeholder.greedy'
+   --> /mod.ts:107:22
+    | 
+107 | placeholder.greedy = greedy;
```

Comment:
We can't transform:

```ts
export function test() {
}

test.greedy = greedy;

export function greedy() {
}
```

To:

```ts
export function test() {
}

export namespace test {
  export var greedy = greedy; // notice issue here
}

export function greedy() {
}
```

...so we just raise a diagnostic.

Reply:
There seems to be quite a few examples of this in the ecosystem tests - oh well we can address in a follow up.